genesyscloud/resource_genesyscloud_architect_schedules_test.go

```diff
@@ -41,6 +41,7 @@ func TestAccResourceArchitectSchedules(t *testing.T) {
 					resource.TestCheckResourceAttr("genesyscloud_architect_schedules."+schedResource1, "start", start),
 					resource.TestCheckResourceAttr("genesyscloud_architect_schedules."+schedResource1, "end", end),
```

Comment:
Could you add a test case creating a schedule with a division (genesyscloud_auth_division) to confirm that this change works?

Reply:
Added a test case in recent commit.